schema.sql

```diff
@@ -8,4 +8,20 @@ CREATE TABLE animals (
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL(7,2) NOT NULL);
 
-ALTER TABLE animals ADD COLUMN species VARCHAR(30);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species VARCHAR(30);
+
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    full_name VARCHAR(30) NOT NULL,
+    age INT,
+    PRIMARY KEY(id));
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(20) NOT NULL);
+ALTER TABLE animals ADD CONSTRAINT pk_animals PRIMARY KEY (id);
+ALTER TABLE species ADD CONSTRAINT pk_species PRIMARY KEY (id);
+
+ALTER TABLE animals DROP COLUMN IF EXISTS species;
+ALTER TABLE animals ADD COLUMN  species_id INT;
+ALTER TABLE animals ADD COLUMN owner_id INT;
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id);
```

Comment:
- [ ] Great job so far creating your tables. However, I would like to suggest that you add a foreign key referencing the `owners` table as mentioned in the project requirements.
<img width="661" alt="Screenshot 2023-09-01 at 08 31 26" src="https://github.com/Grandi0z/vet_clinic_database/assets/107192076/6e7b9ef8-6851-4cd9-8858-069f8cce2b45">